### PR TITLE
[stable-9] Enable Radeon and Nouveau kernel modules in installer image

### PR DESCRIPTION
**Pull request by Tmcg2**
In order for AMD and Nvidia cards to display during the installer we need
to have the right kernel modules loaded. Turn on the approparite defconfig
options.

Install the approrpiate libdrm and linux firmware packages for radeon and
nvidia cards as well

Tested with an AMD W2100 and Nvidia K2200 on an HP EliteDesk 800 G3

OXT-1600

Signed-off-by: Tyler McGavran <mcgavrant@ainfosec.com>

### CODE REVIEW

**Comment by crogers1** [comment]
Does this also fix the
```
mkdir -p /mnt/part2/dom0/sys/kernel/security
mount -o bind /sys/kernel/security /mnt/part2/dom0/sys/kernel/security
cp: error writing '/tmp/tmp.xss0ksPq2v/part1/kcmdline': No space left on device
cp: error writing '/tmp/tmp.xss0ksPq2v/part1/a
```
error that was showing up too?

Also please edit the PR title and add "STABLE-9:" to it, also please open a master PR as well.

File: recipes-core/images/xenclient-installer-image.bb [hunk 1]
```diff
@@ -34,7 +34,9 @@ BAD_RECOMMENDATIONS += " \
     ${@bb.utils.contains('IMAGE_FEATURES', 'web-certificates', '', 'ca-certificates', d)} \
 "
 
-IMAGE_FEATURES += "empty-root-password"
+IMAGE_FEATURES += "\
+    empty-root-password \
```
Comment (crogers1) [inline recipes-core/images/xenclient-installer-image.bb]: This change seems unneeded, can you take it out?

**Comment by Tmcg2** [comment]
I did not look into the "no space left issue", but I can certainly try doing a full install a couple times and see if the issue rears its head.

**Comment by Tmcg2** [comment]
After doing a full install five times with an AMD W2100 I have seen the "no space left" error once and successfully installed the other four times

**Comment by jandryuk** [comment]
@Tmcg2 looks good.

One small git thing.  We usually create the commit on a branch off master.  Then for the release branch, we use `git cherry-pick -x` to add the commits.  That adds `(cherry picked from commit .......)` so you know one is the cherry-pick of the other.

For example, see https://github.com/OpenXT/xenclient-oe/pull/1152/commits/3e8ea24b087670d489222eafda9ccd273e907d3d and https://github.com/OpenXT/xenclient-oe/pull/1162/commits/ba8f880432631bf197e3a9cf101c6189c75eb534

```
git checkout -b s9-feature stable-9
git cherry-pick -x feature
```

Without -x, you don't get `(cherry picked from commit .......)`.  I've added an alias in my .gitconfig to include it and shorten the command:
```
[alias]
	cpx = cherry-pick -x
```

**Comment by Tmcg2** [comment]
Good to know, I'll keep that in mind for future PRs

**Comment by Tmcg2** [comment]
I have continued to try and repro the 'no space left' issue in the linked OXT ticket and I've not see it a second time. My suspicion now is that the one time I reported earlier was me using an un-patched installer and mistakenly thinking that I was using the installer in this PR

**Comment by jean-edouard** [comment]
Merging soon.

**Comment by tijko** [comment]
Tested [Build-6573](http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6573) on a **HP EliteDesk 800 GZ TWR**

With this hardware under UEFI I was able to verify,

  1. With the boot device set to onboard graphics device, install succeeds and boots
  2. With the boot device set to PCI, the installer warning appears and install succeeds but the boot device has to be switched back to onboard or startup fails.

**Pull request merged**